README.md

```diff
@@ -129,14 +129,19 @@ compatibility.
 If you wish to set your system up to forward logs to a set of remote log
 servers, in either the server or client case, you should use the following:
 
-```ruby
-include '::simp_rsyslog::log_shipper'
+```yaml
+simp_rsyslog::forward_logs: true
 ```
-
 This will use the `$simp_options::syslog::log_servers` and
 `$simp_options::syslog::failover_log_servers` variables to set the targets for
 your logs. Alternatively, you can specify the targets in Hiera directly.
 
+If your system is a logserver make sure it is not in the list defined by
```

Comment:
I'm not understanding.  Are you trying to discern the case in which the host is a log server?

Reply:
This is already described in the warning text block below and is not needed.